test/lib/apigw.test.js

```diff
@@ -0,0 +1,204 @@
+const { expect } = require("@oclif/test");
+const AWSMock = require("aws-sdk-mock");
```

Comment:
can we use jest to mock the aws sdk instead since I already use it in the rest of the rests?